backend/clickhouse/query.go

```diff
@@ -157,10 +158,11 @@ func readObjects[TObj interface{}](ctx context.Context, client *Client, config m
 	sql, args := sb.BuildWithFlavor(sqlbuilder.ClickHouse)
 
 	span, _ := util.StartSpanFromContext(ctx, "clickhouse.Query")
-	span.SetAttribute("Table", innerTableConfig.TableName)
-	span.SetAttribute("Query", sql)
-	span.SetAttribute("Params", params)
-	span.SetAttribute("db.system", "clickhouse")
+	span.SetAttribute(string(semconv.DBSQLTableKey), innerTableConfig.TableName)
+	span.SetAttribute(string(semconv.DBStatementKey), sql)
+	span.SetAttribute(string(semconv.DBSystemKey), "clickhouse")
+	span.SetAttribute("db.statement_params", params)
```

Comment:
this isn't a semconv variable?

Reply:
Didn't see one at first, but now I see [`[db.operation.parameter.<key>`](https://opentelemetry.io/docs/specs/semconv/attributes-registry/db/#db-operation-parameter) which looks like the right thing to use. Will update!

Reply:
Looks like it's not available in the `semconv` library we are using yet, as it's still in the rc stage.

Reply:
I did use that pattern though for emitting all the params on our spans.

Reply:
I couldn't figure out a clean way to map the keys to the arguments, so I just put them all on `db.operation.parameters`. Not part of the spec, but pretty close and thought it made the most sense.

I also added attributes for all the other ClickHouse query spans.